### PR TITLE
Dpe 447 retired form redirects

### PR DESCRIPTION
**Pull request by davidu1975**
Redirects for retired contact forms behind DPE feature flag

as per spreadsheet

https://docs.google.com/spreadsheets/d/1qJ0P_4FNMVhCi6qPc98lIidSMEVCiQGruB3tWuDuSYs

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-1139
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Housekeeping

- [x] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.
- [x] I have checked that my PR is using the latest package versions of: great-components, directory-constants, directory-healthcheck, directory-validators, directory-components, directory-api-client, directory-ch-client, django-staff-sso-client, directory-forms-api-client, directory-sso-api-client, sigauth

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: contact/templatetags/dep_tags.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+register = template.Library()
+
+
+@register.simple_tag()
+def get_contact_breadcrumb_details():
+    if settings.FEATURE_DIGITAL_POINT_OF_ENTRY:
+        return {'url': DIGITAL_ENTRY_POINT_TRIAGE_HOMEPAGE, 'text': 'Help and Support'}
+    else:
+        return {'url': 'contact:contact-us-routing-form', 'text': 'Contact us'}
```
Comment (lewis-coulson-dit) [inline contact/templatetags/dep_tags.py:14]: just need a unit test for this and think we're all good

File: core/cms_slugs.py [hunk 2]
```diff
@@ -16,3 +16,5 @@
 # These ones will be removed when the V1->V2 migration is complete
 CONTACT_URL = '/contact/'
 FEEDBACK_CONTACT_URL = '/contact/feedback/'
+
+DIGITAL_ENTRY_POINT_TRIAGE_HOMEPAGE = '/help-and-support/help-and-support/'
```
[the remark this answers is not in the document]
Comment (lewis-coulson-dit) [reply core/cms_slugs.py:20]: could you please change to '/support/help-and-support/'

File: contact/templatetags/dep_tags.py (hunk 1 above)
Comment (davidu1975) [reply contact/templatetags/dep_tags.py:14]: added 2 tests, need 'url' in correct format

File: core/cms_slugs.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (davidu1975) [reply core/cms_slugs.py]: done

**Review by lewis-coulson-dit — APPROVED** [review]

**Pull request merged**